src/components/table.js

```diff
@@ -58,6 +58,50 @@ const activityColumns = (object)=>[
             }
         }
     },
+    ...object=='repo'?
+        [{
+            title: t('大屏'),
+            dataIndex: 'name',
+            align:'center',
+            width: '20%',
+            //根据行号判断，前300的项目可点击大屏
+            render: function (text, row, index) {
+                //console.log('andy',row);
+                let [org_name,repo_name]=text.split("/");
+                //console.log('year',year);
+                //console.log('month',month);
+                let myyear=year;
+                let mymonth=month;
+                switch (mymonth) {
+                    case 0:
+                    {
+                        myyear--;
+                        mymonth='12';
+                        break;
+                    }
+                    case 10:
+                    {
+                        mymonth='10';
+                        break;
+                    }
+                    case 11:
+                    {
+                        mymonth = '11';
+                        break;
+                    }
+                    default:
+                        mymonth='0'+month;
+                        break
+                }
+                //console.log('myyear',myyear);
+                //console.log('mymonth',mymonth);
+                let param='{\"org_name\":\"'+org_name+'\",\"repo_name\":\"'+repo_name+'\",\"t_month\":\"'+myyear+'-'+mymonth+'-01\"}';
```

Comment:
2. using [JS template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to substitute `+` can make your life easier.